conn.go

```diff
@@ -30,6 +30,11 @@ type Conn struct {
 	// indicating server name, status, data policy and capabilities.
 	Greeting
 
+	// LoginResult holds the last received login response message's Result
+	// from the server, which for some servers might include diagnostics such
```

Comment:
…for which some servers…

Reply:
meh, you're parsing it into a different grammatical construct, mostly because I was missing a pair of commas I needed for my construct.  Okay, will fix. :)